tools/pythonpkg/tests/fast/spark/test_spark_to_parquet.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import tempfile
+
+import os
+
+_ = pytest.importorskip("duckdb.experimental.spark")
+
+from duckdb.experimental.spark.sql import SparkSession as session
```

Comment:
this is unused, no?

Reply:
Ah yes, there must be something overriding my normal VSCode settings as normally that would point this out: `fcea55801ce6e13e7962be7f424236f9e7fe1154`